include/CL/sycl/math.hpp

```diff
@@ -0,0 +1,159 @@
+#ifndef TRISYCL_SYCL_MATH_HPP
+#define TRISYCL_SYCL_MATH_HPP
+
+/** \file
+    Implement a wrapper to OpenCL math operations
```

Comment:
"wrapper around"? Ask some native English speakers in the lab. :-)
I guess it is not only for OpenCL